src/logic/contracts/getSafeCreationGasEstimate.tsx

```diff
@@ -3,6 +3,7 @@ import { fromTokenUnit } from 'src/logic/tokens/utils/humanReadableValue'
 import { formatAmount } from 'src/logic/tokens/utils/formatAmount'
 import { getNetworkInfo } from 'src/config'
 import { calculateGasPrice } from 'src/logic/wallets/ethTransactions'
+import { throttle } from 'lodash'
```

Comment:
In the past, to avoid packaging the entire lib, you had to import functions like so:
```
import throttle from 'lodash/throttle'
```
Dunno if they fixed it and it's now correctly tree-shaked.